llama_stack/providers/remote/vector_io/milvus/milvus.py

```diff
@@ -92,7 +127,17 @@ async def query_keyword(
         k: int,
         score_threshold: float,
     ) -> QueryChunksResponse:
-        raise NotImplementedError("Keyword search is not supported in Milvus")
+        # Simple text search using content field
```

Comment:
Can we add validation to raise that keyword search is not yet supported for inline milvus? 

Here's the issue in the Milvus repo to track https://github.com/milvus-io/milvus/issues/40848

As discussed, I don't recommend adding integration tests for the remote container yet as it'll just add more maintenance to the actions. Once milvus lite supports it, it'll be sufficient to use regular unit tests so I think this is a rational choice.